extensions/jni/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ file(COPY ${JNI_FRAMEWORK_JAR_SOURCE} DESTINATION ${JNI_FRAMEWORK_JAR_BIN})
 file(COPY ${JNI_ASSEMBLY_JAR_SOURCE} DESTINATION ${JNI_ASSEMBLY_JAR_BIN})
 
 message("Building NiFi assembly for packaging ....")
-execute_process(COMMAND "mvn" "package" "-Denforcer.skip=true"
+execute_process(COMMAND "mvn" "-q" "package" "-Denforcer.skip=true"

```

Comment:
Could potentially be a "-B" for batch with more output, but it provides little value in the context of the core cpp build